hw/ip/aes/doc/_index.md

```diff
@@ -149,8 +166,13 @@ The AES unit operates as follows:
     4. AddRoundKey Transformation: The round key is XORed with the output of the MixColumns operation and stored back into the State register.
        The 128-bit round key itself is extracted from the current value in the Full Key register.
        In parallel, the full key used for the next round is computed on the fly using the key expand module.
-1. Finally, the AES unit performs the final encryption/decryption round in which the MixColumns operation is skipped.
-   The output of this final round is forwarded to the output register in the CSRs and not stored back into the State register.
+    If running in CTR mode, the counter module iteratively updates the IV in parallel to the cipher core performing encryption/decryption.
+    Internally, the counter module uses one 16-bit counter, meaning it requires 8 clock cycles to increment the 128-bit counter value stored in the IV register.
```

Comment:
incrementers are pretty cheap, why one 16b vs 32b or 128b? There's added complexity to
interlock the increment. Also, would it take a 8 cycles if there is no carry, or stop when no
carry?

Reply:
follow up: what is the impact on throughput?

Reply:
There is no impact on throughput. One block takes 12/14/16 cycles. We start the increment after having read the counter value and one increment 8 cycles. I clarified this in the doc.

I just thought it would be a waste to have it increment in 1 cycle if it is not absolutely needed. But I can change this.

Reply:
makes sense. fine to keep small incrementer, I leave it to you to implement.
I will say it does require DV testing of corner cases. what happens if SW
disables during the IV increment period, for instance? I'll let @rasmus-madsen 
answer if there is concern about DV complexity.